foodnet/membership/views.py

```diff
@@ -1,15 +1,76 @@
 # -*- coding: utf-8 -*-
-from django.shortcuts import render
-
 import logging
 
+from django.conf import settings
+from django.shortcuts import render, redirect
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.decorators import login_required
+from django.core.urlresolvers import reverse
+from django.contrib import messages
+
+from django.views.decorators.cache import never_cache
+from django.views.decorators.csrf import csrf_protect
+from django.views.decorators.debug import sensitive_post_parameters
+
+from .forms import LoginForm
+
 
 log = logging.getLogger(__file__)
 
 
+@sensitive_post_parameters()
+@csrf_protect
```

Comment:
iirc, this is reduntant, on by default, unless CRSF is explicitly disabled: https://docs.djangoproject.com/en/1.8/ref/csrf/#how-to-use-it